app/javascript/src/components/editor/Modals/TranslationKeysModal.svelte

```diff
@@ -28,6 +32,109 @@
     selectedKey = value
     newKeyInput.value = ""
   }
+
+  const DELIMITER_TO_FILE_TYPE = new Map([
+    [",", "CSV"],
+    ["\t", "TSV"],
+    [null, "TSV/CSV"]
+  ])
+
+  async function importKeysFromClipboard() {
```

Comment:
Functions should have a return type (There should be a linter for this, but it's possible the linter doesn't work in Svelte files)

```suggestion
  async function importKeysFromClipboard(): void {
```